hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -908,105 +897,105 @@ struct ud2 {};
  * Arithmetic instructions.
  */
 // add: s0 + {s1|m} => {d|m}, sf
-struct addl   { Vreg32 s0, s1, d; VregSF sf; };
-struct addli  { Immed s0; Vreg32 s1, d; VregSF sf; };
-struct addlm  { Vreg32 s0; Vptr m; VregSF sf; };
-struct addlim { Immed s0; Vptr m; VregSF sf; };
-struct addq  { Vreg64 s0, s1, d; VregSF sf; };
-struct addqi { Immed s0; Vreg64 s1, d; VregSF sf; };
-struct addqim { Immed s0; Vptr m; VregSF sf; };
+struct addl   { Vreg32 s0, s1, d; VregSF sf; uint8_t fl; };
```

Comment:
I would abstract away the underlying representation here with `using Vflags = uint8_t`, if the idea is to make it more generally usable (we might want to make it a union of per-arch information, etc.)